### PR TITLE
Hovercard fix

### LINKED ISSUE #9132: Hover card jump
### Environment Information 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

#### Actual behavior:
 
The hover card is "jumping" if user move mouse into the card when the card is rendering
fabric_hovercard_jump

Reproable on fabric display site: https://developer.microsoft.com/en-us/fabric#/controls/web/hovercard 
 
#### Expected behavior:
 
The card should not "jump" when it is rendering

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by sopranopillow**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9132
- [x] Include a change request file using `$ npm run change`

#### Description of changes

There are two changes, one was that a setState was being executed more than it needs to so I moved it to the inside of the if statement. The second issue is what fixes the jump, I added an if statement to check if the card has a final height and a position since once you have both there is no need to keep updating the position.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9501)

### CODE REVIEW

File: common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Moved setSate in _setHeightOffsetEveryFrame so it does not run more than necessary and added and if to check if there is a given position and final height, then stop running _updateAsyncPosition",
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json]: nit: comment in here must be what has been fixed in what component with a short sentence. It helps scanning through release notes. Start with something like: `CalloutContent: fixing bug....`. No need implementation details.

**Comment by natalieethell** [comment]
@joschect you have more context on Callout here - could you take a look when you have a moment?

**Comment by joschect** [comment]
@natalieethell Sure can! I spoke with @sopranopillow offline a bit about this already. There is a bug somewhere deep in the positioning logic that breaks the hovercard when coverTarget is set to true. That's currently being investigated!

File: common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json (hunk 1 above)
Comment (dzearing) [reply common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json]: It should read like release notes. So what behavior difference does the user care about reading?

File: packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx [hunk 2]
```diff
@@ -105,7 +105,9 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
       if (!this._hasListeners) {
         this._addListeners();
       }
-      this._updateAsyncPosition();
+      if (!this.state.positions || !this.props.finalHeight) {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx]: add a comment for what's going on here. Why is this needed?

File: common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json (hunk 1 above)
Comment (dzearing) [reply common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json]: @sopranopillow ping on this!

**Comment by dzearing** [comment]
Closing per discussion.
